hack/deploy_csv.yaml

```diff
@@ -54,20 +54,6 @@ data:
               spec:
                 description: NSTemplateSetSpec defines the desired state of NSTemplateSet
                 properties:
-                  clusterResources:
```

Comment:
I need to verify why this was removed.

Reply:
did you generate the files while having api repo in master branch and with the latest commits pulled from the origin?

Reply:
yes, but I regenerated these files using the `make generate-olm-files` command in the `member-operator` repository. But can this explain the problem?

Reply:
hmm, running the `make generate` from the `codereadytoolchain/api` repo restored these `clusterResources` properties.

Reply:
fixed in https://github.com/codeready-toolchain/member-operator/pull/132/commits/4ff21fe290068148ce95301340bde952057fd08b

Reply:
hmm... weird. The `make generate-olm-files` should not take anything from the `api` repo yet :man_shrugging: (except for the script)

I tried it locally it it works fine.

Reply:
indeed, I re-ran the `make generate-olm-files` and I did not get any change, which means that the `clusterResources` properties were not removed again. I have no idea what happened before 🤨